lib/Watcher.php

```diff
@@ -75,14 +80,17 @@ class Watcher {
 	 * @param FaceMapper $faceMapper
 	 * @param ImageMapper $imageMapper
 	 * @param PersonMapper $personMapper
+	 * @param FaceManagementService $faceManagementService
 	 */
-	public function __construct(IConfig       $config,
-	                            ILogger       $logger,
-	                            IDBConnection $connection,
-	                            IUserManager  $userManager,
-	                            FaceMapper    $faceMapper,
-	                            ImageMapper   $imageMapper,
-	                            PersonMapper  $personMapper)
+	public function __construct(
```

Comment:
I told you I would never demand changes styles.. Just comment.. :wink: 
The way to avoid these bad tabulations, is to use a mix of tab and spaces .. :sweat_smile:

Reply:
fixed in version 2